expression/builtin_miscellaneous.go

```diff
@@ -0,0 +1,445 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+// // Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expression
+
+import (
+	"github.com/juju/errors"
+	"github.com/pingcap/tidb/context"
+	"github.com/pingcap/tidb/util/types"
+	"time"
+)
+
+var (
+	_ functionClass = &sleepFunctionClass{}
+	_ functionClass = &lockFunctionClass{}
```

Comment:
Maybe we'll never implement this kind of function.
what's your opinion? @shenli